### PR TITLE
[HDX-8327] more automation and fixes.

### PR DESCRIPTION
**Pull request by teodorescuserban**
Completed.

### CODE REVIEW

**Comment by teodorescuserban** [comment]
@alexandru-m-g i think it's ready. take it for a spin :)

**Review by danmihaila — COMMENTED** [review]
it looks good for me. @alexandru-m-g  ?

File: requirements-hdxckantool.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+beautifulsoup4==4.9.1
+awscli==1.19.112
```
Comment (alexandru-m-g) [inline requirements-hdxckantool.txt:2]: In the end is this a version of aws cli that runs with the current HDX dependencies ?

Comment (teodorescuserban) [reply requirements-hdxckantool.txt:2]: Yes, @alexandru-m-g; and `beautifulsoup4` is the same version required by dev requirements so it doesn't break those either.

**Pull request merged**